auth-cas/config.php

```diff
@@ -61,15 +61,33 @@ function getOptions() {
         'label' => $__('Service label'),
         'configuration' => array('size'=>60, 'length'=>100),
         'hint' => $__('The text "Login with {label}" will appear on the login
-            button. By default this is "CAS".'))),
+          button. By default this is "CAS".'))),
       'cas-name-attribute-key' => new TextboxField(array(
         'label' => $__('Name attribute key'),
-        'configuration' => array('size'=>60, 'length'=>100))),
+        'configuration' => array('size'=>60, 'length'=>100),
+        'hint' => $__('This maps to the "name" form field.'))),
       'cas-email-attribute-key' => new TextboxField(array(
         'label' => $__('E-mail attribute key'),
-        'configuration' => array('size'=>60, 'length'=>100))),
+        'configuration' => array('size'=>60, 'length'=>100),
+        'hint' => $__('This maps to the "email" form field.'))),
+      'cas-custom-attributes' => new TextareaField(array(
+        'label' => $__('Custom attributes'),
+        'configuration' => array('cols'=>60, 'rows'=>5, 'length'=>1000,
+          'html'=>false, 'placeholder'=>
+            "sAMAccountName,adusername\nmobile,mobile"),
```

Comment:
If you don't change the placeholder, does that value end up being the default?

Reply:
No, placeholder does not affect value.